src/test/scala/viper/gobra/ExprTypingUnitTests.scala

```diff
@@ -18,7 +18,10 @@ class ExprTypingUnitTests extends FunSuite with Matchers with Inside {
 
   test("TypeChecker: should classify an integer literal as integer") {
     frontend.exprType(PIntLit(42))() should matchPattern {
-      case Type.IntT =>
+      // the type should be Type.Int instead of Type.UntypedConst due to the implementation of
+      // front.end(exprType) which in practice evaluates the type of the literal in a
```

Comment:
is the function called "front.end". This seems like a spelling error.

Reply:
Fixed in 8cb139e